src/toil/leader.py

```diff
@@ -290,9 +295,11 @@ def processRemovedJob(jobStoreID):
                 else:
                     raise
             if jobWrapper.logJobStoreFileID is not None:
-                logger.warn("The jobWrapper seems to have left a log file, indicating failure: %s", jobStoreID)
                 with jobWrapper.getLogFileHandle( self.jobStore ) as logFileStream:
-                    logStream( logFileStream, jobStoreID, logger.warn )
+                    messages = logFileStream.read().splitlines()
+                logFormat = '\n%s    ' % jobStoreID
+                logger.warn('The jobWrapper seems to have left a log file, indicating failure: %s\n%s',
```

Comment:
`job wrapper` or just `job` since wrapper vs. job distinction is insignificant here.